src/pal/src/misc/time.cpp

```diff
@@ -336,62 +332,25 @@ PALAPI
 ULONGLONG
 GetTickCount64()
 {
-    ULONGLONG retval = 0;
+    LONGLONG retval = 0;
```

Comment:
This one doesn't have a direct correspondence in CoreFX, but it's also simpler and less error prone to just implement it in terms of `QPC`/`QPF`. It's also not likely worth micro-optimizing to avoid these calls since it is only measuring time in milliseconds.

Reply:
This function had one difference - on contrary to the QueryPerformanceCounter, it was using HAVE_CLOCK_MONOTONIC_COARSE for performance reasons. It was added in https://github.com/dotnet/coreclr/pull/2293 to fix #2257.
I would like to preserve this performance benefit.

Reply:
Will update....